app/views/cases/offender_sar/_data_requests.html.slim

```diff
@@ -32,14 +32,16 @@
                 = data_request.cached_num_pages
               td
                 = data_request.status
-              td
-                = link_to t('cases.data_requests.edit.link'), edit_case_data_request_path(case_details, data_request), class: 'data-requests__action'
+              - if can_be_edit
+                td
+                  = link_to t('cases.data_requests.edit.link'), edit_case_data_request_path(case_details, data_request), class: 'data-requests__action'
 
           - if case_details.data_requests.many?
             = render partial: 'shared/table_total_row', locals: { total: case_details.data_requests.sum(:cached_num_pages), label_span: '3', value_span: '3' }
 .data-request-buttons
   - if case_details.current_state != 'closed'
     = action_button_for(:record_data_request)
```

Comment:
When testing this locally linking to an open offender sar, this method blows up because @case isn't set in our controller but it's assumed from the cases_helper button thingies. 
![image](https://user-images.githubusercontent.com/1161161/99452391-5dd49200-291b-11eb-9aba-14e3ae4ef9ee.png)

Is it worth doing a feature spec example on an open offender sar as well as a closed one?

Reply:
thank you for catching this bug, I don't know why I didn't try the new offender sar case -_-,  I will not show the new-data-request button if it from complaint case .